pkg/virt-handler/device-manager/deviceplugin/v1beta1/BUILD.bazel

```diff
@@ -1,41 +1,21 @@
-package(default_visibility = ["//visibility:public"])
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_library",
-)
+# gazelle:ignore
```

Comment:
we copied `device-manager` pkg from some external sources, so `proto` target misses some dependencies for the proper build process. To avoid this problem I build `go` target without `proto` middle target and I add `# gazelle:ignore` to avoid re-generation of the `BUILD.bazel` file.

Reply:
ok, got it